R/utils.R

```diff
@@ -109,8 +109,8 @@ create_stressdata_masterdata_file_paths <- function(data_prep_timestamp, twodii_
   path_parent <- path_dropbox_2dii("PortCheck", "00_Data", "07_AnalysisInputs", data_prep_timestamp)
 
   paths <- list(
-    "stress_test_masterdata_debt.rds",
-    "stress_test_masterdata_ownership.rds",
+    "prewrangled_financial_data_bonds.rds",
+    "prewrangled_financial_data_equity.rds",
     "stress_test_masterdata_credit.rds"
```

Comment:
why not also update it for credit? if not let's set up a follow up task

Reply:
I did not update if for credit yet, as there is no prewrangling code around in the code yet, so I did not know how to prewrangle it.
If it is exactly the same steps as for bonds and eq I am happy to do that and to also extend the valid asset types in read_company_data. If not I would prefer to put both aspects in a follow up :)

Reply:
fair point